### ISSUE TITLE
swig cannot parse CGAL/version.h

### ISSUE THREAD

**Issue by xantares** [CGAL/cgal#4355]
swig fails to parse version.h:
```
#define CGAL_VERSION 4.14.2
#define CGAL_VERSION_NR 1041421000
#define CGAL_SVN_REVISION 99999
#define CGAL_GIT_HASH 19efe6e568f8d9cf090193b029b143e769b3ea18
#define CGAL_RELEASE_DATE 20190329
```
it chokes on the CGAL_GIT_HASH define with ":EOF: Error: Exponent does not have any digits"
I'll ask swig (https://github.com/swig/swig/issues/1666), but you may want to add quotes here

The version number may be quoted too to avoid the "Bad constant value" warning

**Comment by lrineau** [comment]
@xantares Which version of SWIG has the issue? I am wondering if your patch in https://github.com/CGAL/cgal/pull/4358 should be patchported to the bug-fix releases branches (for 4.14.3 and 5.0.1).

**Comment by xantares** [comment]
only swig 4.x is affected.
imho it doesnt hurt to backport.
